### PR TITLE
implemented search and tests

### PR DESCRIPTION
**Pull request by mariiahorbova**

### CODE REVIEW

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]
tests failed

File: tests/test_view.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_login_required_manufacturers_view(self):
+        response = self.client.get(reverse("taxi:manufacturer-list"))
+
+        self.assertNotEqual(response.status_code, 200)
+
+
+class PrivateViewsTests(TestCase):
```
Comment (YuriiKindrat) [inline tests/test_view.py:24]: what about test for toggle assign to the car feature ?

File: templates/taxi/car_list.html [hunk 2]
```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="🔍">
+  </form>
```
Comment (YuriiKindrat) [inline templates/taxi/car_list.html:14]: could u make this as includes to use it in all ur templates ?

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: templates/includes/search.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{% load crispy_forms_filters %}
+<form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="🔍">
+</form>
```
Comment (YuriiKindrat) [inline templates/includes/search.html:5]: no blank line ((

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 🐣

**Pull request closed without merging**